development/views.py

```diff
@@ -135,3 +138,14 @@ def form_valid(self, form):
             )
             self.success_url = reverse_lazy('development:addbot')
         return super().form_valid(form)
+
+
+@require_http_methods(["GET"])
+def delete_bot(request, pk):
```

Comment:
This should not be a GET. Ideally this is a DELETE but you can use a POST also

Reply:
Done!